po4a-update-templates.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+# You need po4a > 0.54, see https://github.com/mquinson/po4a/releases
+# There is no need of system-wide installation of po4a
+# Usage: PERLLIB=/path/to/po4a/lib make_pot.sh
+# you may set following variables
+# SRCDIR root of the documentation repository
+# POTDIR place where to create the .pot templates
+
+
+####################################
+# INITIALISE VARIABLES
+####################################
+
+# root of the documentation repository
+SRCDIR_MODULE="./wiki/en"
+
+# place where to create the .pot template files
+if [ -z "$POTDIR" ] ; then
+    POTDIR="./translator-files/l10n/templates"
+fi
+
+# place where the po files are
+if [ -z "$PO_DIR" ] ; then
+	PO_DIR="./translator-files/l10n/po"
+fi
+
+####################################
+# TEST IF IT CAN WORK
+####################################
+
+# Check if po4a is installed
+if ! [ -x "$(command -v po4a)" ] ; then
+    echo 'Error: please install po4a.' >&2
+    exit 1
+fi
+
+if [ ! -d "$SRCDIR_MODULE" ] ; then
+    echo "Error, please check that SRCDIR matches the root of the documentation repository"
+    echo "Your specified modules are in $SRCDIR_MODULE"
+    exit 1
+fi
+
+####################################
+# Process the documents
+####################################
+
+while IFS= read -r -d '' file
+do
+    basename=$(basename -s .md "$file")
+    dirname=$(dirname "$file")
+    path="${dirname#$SRCDIR_MODULE/}"
+
+    if [ "$dirname" = "$SRCDIR_MODULE" ]; then
+        potname=$basename.pot
```

Comment:
@ignotus666 I'm a bit worried about quoting here. If $basename includes spaces it might break.

Reply:
Done